ibm/mas_devops/playbooks/cp4d/install-services-db2.yml

```diff
@@ -2,16 +2,24 @@
 - hosts: localhost
   any_errors_fatal: true
   vars:
+    # Cluster details
+    # ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+    cluster_name: "{{ lookup('env', 'CLUSTER_NAME') }}"
```

Comment:
Why do we need the CLUSTER_NAME and CLUSTER_TYPE for installing db2 service?

Reply:
We can probably remove name, but I'm using 'cluster_type' for now as a check to verify whether it's a roks cluster or not. CPD v4 install has specific roks steps to reload worker nodes that won't work for non-roks clusters

Reply:
> CPD v4 install has specific roks steps to reload worker nodes after we setup image pull secret that won't work for non-roks clusters

That sounds horrible, that can't be their normal install process surely!!  But that's a question/topic for another day to take up with the CP4D team.